clang/lib/Sema/SemaOverload.cpp

```diff
@@ -14027,6 +14027,11 @@ Sema::BuildCallToObjectOfClassType(Scope *S, Expr *Obj,
 
   // If this is a variadic call, handle args passed through "...".
   if (Proto->isVariadic()) {
+    // Diagnose variadic calls in SYCL.
+    if (getLangOpts().SYCLIsDevice && !isUnevaluatedContext())
```

Comment:
Well, there isn't really a 'decl' to check!  This is the operator call for a type, so the decl doesn't actually necessarily exist (and we don't actually have it).  Since we aren't marking call operators a 'known good' there is no reason to check it.

Reply:
Oh, okay then